op-devstack/sysgo/system.go

```diff
@@ -196,6 +196,66 @@ func DefaultInteropSystem(dest *DefaultInteropSystemIDs) stack.Option[*Orchestra
 	return opt
 }
 
+func DefaultInteropProofsSystem(dest *DefaultInteropSystemIDs) stack.Option[*Orchestrator] {
```

Comment:
Is there any reason all tests wouldn't just use this? It doesn't seem great that we're testing the interop hard fork against a system that hasn't had contracts migrated to interop like they actually would be anyway.

Reply:
Performing the OPCM migration takes about 60s to run. It's fast but not that fast and I want to incur this setup cost only for proofs-related tests. There are many more tests that do not interact with the lockbox or proof system in anyway that don't benefit from the OPCM migrate.

Reply:
Oof. 60 seconds really hurts. Seems like we need to prioritise deploying multiple new chains into a shared dependency set higher than I thought so our test setup reflects reality.